tools/releaseBuild/azureDevOps/AzArtifactFeed/PSGalleryToAzArtifacts.yml

```diff
@@ -7,19 +7,18 @@ resources:
 queue:
```

Comment:
@TravisEz13, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of 'jedexcup.cgd'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 208
208:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```